lxd/main_checkfeature.go

```diff
@@ -330,3 +363,7 @@ func canUseSeccompListener() bool {
 func canUseSeccompListenerContinue() bool {
 	return bool(C.seccomp_notify_aware == 2)
 }
+
+func canUsePidFds() bool {
+	return bool(C.pidfd_aware)
```

Comment:
Shouldn't this also confirm liblxc support?

Reply:
Not really, that's done dynamically. liblxc is smart enough to only allocate a pidfd when it knows the kernel supports it so we can infer liblxc support from InitPidFd() right before we make the call.

Reply:
Aren't we wasting time by always hitting go-lxc for pidfd then either getting told not-supported from that or from liblxc and then hitting the fallback interface?

For other liblxc features we do the detection once at startup and then use that result for all future calls.

Reply:
I think we're doing that for seccomp. You need to merge a small fix for liblxc first, though.